projects/gitea/build.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mkdir $GOPATH/src/code.gitea.io
+mv $SRC/gitea $GOPATH/src/code.gitea.io/
```

Comment:
Why not "go get" in Dockerfile instead of this mkdir, mv

Reply:
When installing the dependencies with "go get" gitea expects packages from code.gitea.io, hence gitea is renamed and moved to that location prior to installing the dependencies. 
I could check if this can be done in the Dockerfile if needed.

Reply:
no worries, leave it then. not a big deal.